crates/l2/contracts/bin/deployer/cli.rs

```diff
@@ -282,6 +282,27 @@ pub struct DeployerOptions {
         help = "Address of the owner of the SequencerRegistry contract, who can upgrade the contract."
     )]
     pub sequencer_registry_owner: Option<Address>,
+    #[arg(
+        long,
+        env = "ETHREX_ON_CHAIN_PROPOSER_INCUSION_MAX_WAIT",
+        help_heading = "Deployer options",
+        help = "Max time in seconds that can pass without enough deposits being processed."
+    )]
+    pub inclusion_max_wait: u64,
+    #[arg(
+        long,
+        env = "ETHREX_ON_CHAIN_PROPOSER_INCLUSION_TARGET",
+        help_heading = "Deployer options",
+        help = "Inclusion target for privileged transactions."
```

Comment:
What does "target" mean here? If it is the amount of txs expected to be included, elaborate on that.